src/utils/cutPlane.js

```diff
@@ -76,128 +73,6 @@ export function getElementBoundingBox(selection) {
     }
   }
 
-  const vertices = Float32Array.from(coordinates)
-  geometry.setAttribute('position', new BufferAttribute(vertices, selection.geometry.index.count))
-  const mesh = new Mesh(geometry)
-  const boundingBox = new Box3()
-  boundingBox.setFromObject(mesh)
   return boundingBox
 }
 
-/**
- * toggleClippingPlane turns clipping plane on and off
- *
- * @param {Boolean } on bouding box
- * @param {Number} expressId centerCoordinates
- * @param {IfcViewerAPI } ifcViewer bouding box
- */
-export function toggleClippingPlane(on, expressId, ifcViewer) {
-  if (on) {
-    const modelCenter = getModelCenter()
-    const boundingBox = getElementBoundingBox(ifcViewer?.IFC.selector.selection.mesh)
-    const selectionAxis = getSelectionAxisFromBoundingBox(boundingBox)
-
-    let direction = 1
-
-    let normal
-    if (
-      selectionAxis.x.size < selectionAxis.y.size &&
-      selectionAxis.x.size < selectionAxis.z.size
-    ) {
-      if (selectionAxis.x.center > modelCenter.x) {
-        direction = -1
-      }
-      normal = new Vector3(direction, 0, 0)
-    } else if (
-      selectionAxis.y.size < selectionAxis.x.size &&
-      selectionAxis.y.size < selectionAxis.z.size
-    ) {
-      if (selectionAxis.y.center > modelCenter.y) {
-        direction = -1
-      }
-      normal = new Vector3(0, direction, 0)
-    } else {
-      if (selectionAxis.z.center > modelCenter.z) {
-        direction = -1
-      }
-      normal = new Vector3(0, 0, direction)
-    }
-
-    const point = new Vector3(
-        selectionAxis.x.center,
-        selectionAxis.y.center,
-        selectionAxis.z.center,
-    )
-
-    ifcViewer?.clipper.createFromNormalAndCoplanarPoint(normal, point)
-    ifcViewer?.IFC.selector.unpickIfcItems()
-  } else {
-    removePlanes(ifcViewer)
-    showElement([expressId], true)
-  }
-}
-
-
-/**
- * toggleClippingPlane turns clipping plane on and off
- *
- * @param {IfcViewerAPI } viewer bouding box
- */
-function removePlanes(viewer) {
```

Comment:
Hey @OlegMoshkovich - is there a reason why we are removing these helper functions? I am using the ``removePlanes`` for the extract level feature.